doc/developer_guide/psyir.rst

```diff
@@ -691,21 +691,35 @@ class, for example:
 From the language-level PSyIR nodes, Container, Routine and Statement have
 the CommentableMixin trait.
 
+Domain-Specific PSyIR
+=====================
+
+The discussion so far has been about generic language-level
+PSyIR. This is located in the ``psyir`` directory and contains nodes,
+symbols, transformations, front-ends and back-ends. None of this is
+domain specific.
+
+To obtain domain-specific concepts the language-level PSyIR can be
+specialised or extended. In LFRic there are specialisations for
+kernel-layer datatypes and symbols. For the algorithm layer in both
+GOcean1.0 and LFRic there are specialisations for invoke's and kernel
```

Comment:
s/invoke's/invokes/ and you could enclose in double backticks if you want (although the trailing 's' is then difficult).

Reply:
I've updated it but left it without the backticks due to the "s" as you point out.